package.json

```diff
@@ -111,7 +111,7 @@
     "font-awesome": "~4.7.0",
     "foundation-sites": "~6.5.3",
     "geckodriver": "^1.19.0",
-    "help-popup": "drugis/help-popup#0.5.1",
+    "help-popup": "drugis/help-popup#1.1.0",
```

Comment:
I feel we could have retagged the new version as the actual 1.0.0 (not like anyone else was using that version) but eh